kselftests/runtest.sh

```diff
@@ -114,8 +114,7 @@ run_test()
 # For upstream kselftest testing, we need a pre-build selftest tar ball url
 install_kselftests()
 {
-	[ ! "$selftests_url" ] && log "no selftests_url" && return 1
-	wget --no-check-certificate $selftests_url -O kselftest.tar.gz
+	wget --no-check-certificate $CKI_SELFTEST_URL -O kselftest.tar.gz
```

Comment:
Not really related to this change specifically (I see it was there from before), but why do we need `--no-check-certificate`? Can't we create a proper certificate with Let's Encrypt or something similar?

Reply:
I don't know, should we create the certification on the machine setup stage?